_posts/en/newsletters/2019-01-08-newsletter.md

```diff
@@ -0,0 +1,168 @@
+---
+title: 'Bitcoin Optech Newsletter #28'
+permalink: /en/newsletters/2019/01/08/
+name: 2019-01-08-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter announces a new maintenance release of Bitcoin
+Core, describes continued discussion about new signature hashes, and
+links to a post about possible economic barriers to LN payments crossing
+different currencies.  Descriptions of notable code changes in popular
+Bitcoin infrastructure projects are also provided.
+
+## Action items
+
+- **Upgrade to Bitcoin Core 0.17.1:** this new [minor][maintenance]
+  version released December 25th restores some previously-deprecated
+  functionality to the `listtransactions` RPC and includes bug fixes and
+  other improvements.  Consider reading the [release notes][0.17.1
+  notes] and [upgrading][0.17.1 bin].
+
+## News
+
+- **Continued sighash discussion:** as mentioned in the News section of
+  [Newsletter #25][], developers on the Bitcoin-Dev mailing list
+  discussed how signature hashes could be modified to give transactions
+  access to new capabilities.  Sighashes give spenders the ability to
+  allow their transactions to be modified in specified ways after they
+  are signed---for example, two people who open a payment channel
+  together can use a particular type of sighash to allow either one of
+  them to unilaterally attach additional transaction fees to a channel
+  close transaction.
+
+    The most recent discussion in this thread of almost 70 posts has
+    mostly involved edge cases related to new sighash flags,
+    particularly a BIP118-like `SIGHASH_NOINPUT_UNSAFE`.  As part of the
+    discussion, protocol developer Johnson Lau described an
+    [optimization for Eltoo-based payment channels][lau bip68].  Also
+    [discussed][rm codesep] is whether the `OP_CODESEPARATOR` opcode
+    should be disabled in a script update that supports MAST (e.g. via
+    Taproot).  That opcode is not in common use, but if you plan to use
+    it in future Script versions, you should comment on the thread.
```

Comment:
Should this be in the Action Items section as well?

Reply:
I think it's ok to leave this a bit buried.  We do want people to comment sooner than later, but I think it's ok if nobody comments until there's been an actual formal proposal of a soft fork with a new Script version that doesn't include this opcode.  Besides, my guess is that nobody is using this opcode.